tests/dummy/app/acceptance/controller.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  componentPath: Ember.computed('model.component', function() {
```

Comment:
Could this logic be moved to the `acceptance` route? Since Ember is moving away from controllers I think it would be good forward-thinking.